alonzo/formal-spec/transactions.tex

```diff
@@ -187,15 +168,14 @@ \subsection{Witnessing}
 
 \begin{itemize}
   \item VKey signatures;
-  \item a set of scripts, containing all the scripts (for any purpose) that are needed to \emph{validate} the transaction,
-  \item a set of terms of type $\Data$, which contains all required datum objects; and
-  \item a map of $\Data$ values indexed by $\RdmrPtr$, which includes all the
-  required redeemers.
+  \item a map of scripts, including non-native scripts;
+  \item a map of terms of type $\Data$, containing all required datum objects; and
```

Comment:
even though it's mostly clear, i think it makes sense to specify that these are hash -> pre-image maps